otx/algorithms/detection/adapters/mmdet/models/detectors/custom_maskrcnn_tile_optimized.py

```diff
@@ -171,12 +171,15 @@ def simple_test(self, img, img_metas, proposals=None, rescale=False):
             img_metas (list): image meta data
             proposals (list, optional): proposals. Defaults to None.
             rescale (bool, optional): rescale. Defaults to False.
+            full_res_image (bool, optional): if the image is full resolution or not. Defaults to False.
 
         Returns:
             tuple: MaskRCNN output
         """
-
         keep = self.tile_classifier.simple_test(img) > 0.45
+        if isinstance(full_res_image, bool):
+            full_res_image = [full_res_image]
+        keep = full_res_image[0] | keep
```

Comment:
Could you explain when full_res_image is not bool?

Reply:
For some reason, NNCF doesn't collate items to list. I'm assuming NNCF implemented customised collate_fn. But mmdetection collects `full_res_image` as list